runtime/statemine/src/lib.rs

```diff
@@ -736,15 +736,15 @@ pub type Executive = frame_executive::Executive<
 >;
 
 impl_runtime_apis! {
-	// impl sp_consensus_aura::AuraApi<Block, AuraId> for Runtime {
-	// 	fn slot_duration() -> sp_consensus_aura::SlotDuration {
-	// 		sp_consensus_aura::SlotDuration::from_millis(Aura::slot_duration())
-	// 	}
-
-	// 	fn authorities() -> Vec<AuraId> {
-	// 		CollatorSelection::Collators()
-	// 	}
-	// }
+	impl sp_consensus_aura::AuraApi<Block, AuraId> for Runtime {
```

Comment:
cc @joepetrowski @bkchr all of what we did for aura support was to just add these few lines here.

Reply:
This is not complete. 

Next week when I'm back, I will be merging the aura pr. Then we can do it here properly.